src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  onDelete: (todoId: number) => void,
+  isProcessedIDs: boolean,
```

Comment:
Consider renaming 'isProcessedIDs' to 'isProcessing' for better readability and to reflect the boolean nature of the variable 📖